build/scripts/run_perf.ps1

```diff
@@ -20,3 +20,7 @@ Invoke-WebRequest -Uri http://dotnetci.blob.core.windows.net/roslyn-perf/cpc.zip
 [Environment]::SetEnvironmentVariable("VS150COMNTOOLS", "C:\\Program Files (x86)\\Microsoft Visual Studio\\VS15Preview\\Common7\\Tools", "Process")
 
 ./cibuild.cmd /testPerfRun /release
+if ($LASTEXITCODE -ne 0)
```

Comment:
This is using $LASTEXITCODE here because I don't want this error checking to be sensitive to stderr. $? is a more general error checking construct, and it appeared that cleanup_perf.ps1 could fail with both exceptions and by error code, so I left it alone.